tcmur_cmd_handler.c

```diff
@@ -682,9 +682,19 @@ static int handle_unmap_in_writesame(struct tcmu_device *dev,
 	uint8_t *cdb = cmd->cdb;
```

Comment:
Nit: typo in commit message: `acquired` -> `required`? Technically I don't think the SCSI spec requires it to be anything more than block-aligned, though.

Reply:
Yeah, right, check the spec, didn't find it. Will fix the commit comment.